src/utils.ts

```diff
@@ -129,3 +129,29 @@ export const parseFormatTokens = (format: string): string[] => {
         return token.startsWith('[') && token.endsWith(']') ? token.slice(1, -1) : token
     })
 }
+
+
+/**
+ * Converts a list of strings to a regex string.
+ * It takes possible matching values to be from longest to shortest.  This
+ * prevents the possibility of a match occurring for a value that also
+ * a substring of a larger value that should have matched (e.g., 'abc'
+ * matching when 'abcdef' should have been the match).
+ *
+ * @param toConvert - An array of string containing all the required regex values
+ */
+export const seqToRE = (toConvert: Array<string>): RegExp => {
+    toConvert = toConvert.sort((a, b) => b.length - a.length)
+
+    for (let value of toConvert) {
+        if (value != "") {
+            break
+        } else {
+            return new RegExp("")
+        }
+    }
```

Comment:
suggestion: You can refactor this code to a simpler one.

```js
    // handle toConvert empty
    
    // empty chars '' are sorted at last 
    // if there are at the first index means that there are only empty chars on the array.
    if (toConvert[0] === '')
        return RegExp("")
    }
```

Reply:
updated